distributed3/core.py

```diff
@@ -104,10 +106,7 @@ def read_sync(s):
     while b''.join(bytes[-len(sentinel):]) != sentinel:
         bytes.append(s.recv(1))
     msg = b''.join(bytes[:-len(sentinel)])
-    try:
-        return loads(msg)
-    except:
-        return msg
+    return loads(msg)
 
 
 sentinel = b'7f57da0f9202f6b4df78e251058be6f0'
```

Comment:
Just random.  We need some way to break between messages.  There are other ways, but this way has worked pretty well so far.  The odds that a random bitstring matches this is `2**320`.